<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-NIKNT8S" failures="1" tests="6" name="Guru99_Negative" time="4.263" errors="0" timestamp="2024-11-03T18:38:47 PST">
  <testcase classname="TestMath" name="TestX" time="4.244"/>
  <testcase classname="TestMath" name="TestDifferenceWithDiffrentValues" time="0.001"/>
  <testcase classname="TestMath" name="TestAdditionWithDiffrentValues" time="0.001"/>
  <testcase classname="TestMath" name="TestPrimeWithDiffrentValues" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [false] but found [true]">
      <![CDATA[java.lang.AssertionError: expected [false] but found [true]
at TestMath.TestPrimeWithDiffrentValues(TestMath.java:21)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1597)
... Removed 36 stack frames]]>
    </failure>
  </testcase> <!-- TestPrimeWithDiffrentValues -->
  <testcase classname="SeleniumNewTest" name="TestMathFunctions" time="0.001"/>
  <testcase classname="SeleniumNewTest" name="TestMathFunctions2" time="0.001"/>
</testsuite> <!-- Guru99_Negative -->
